app/controllers/loan_documents_controller.rb

```diff
@@ -0,0 +1,33 @@
+class LoanDocumentsController < ApplicationController
+
+  def index
+    @loan_documents = Document.all
+  end
+
+  def new
+    @loan_document = Document.new
+  end
+  
+  def show; end
```

Comment:
@ericksonmc Given the structure proposed in this PR, there's no show action and this action should be deleted, along with it's associated template.